importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -4926,7 +4926,7 @@ Note* MusicXMLParserPass2::note(const QString& partId,
             else if (_e.name() == "stem")
                   stem(stemDir, noStem);
             else if (_e.name() == "type") {
-                  small = _e.attributes().value("size") == "cue" | _e.attributes().value("size") == "grace-cue";
+                  small = (_e.attributes().value("size") == "cue") || (_e.attributes().value("size") == "grace-cue");
```

Comment:
Hmm, I was suggesting the parenthezise the entire || expression, not their elements